liferay-echo/liferay-portal
# LPS-186753 New segments and experiences selectors in simulation panel

Motivation
=======
Simulate segments and experiences from the simulation panel
[Link to story or ticket](https://issues.liferay.com/browse/LPS-186753)

Proposed Solution
========
We have adde a new component SegmentsAndExperiencesSelector that handles the selection of segments and experiences and updates the iframe with the preview.

Steps to Verify:
----------------
1. Set feature.flag.LPS-186155=true in portal-ext.properties
2. Open the simulation panel in the home page and check that the experience selector in the header when simulation is opened is hidden.
3. Create some segments in People -> Segments and come back to the simulation panel.
4. Check that the segments selector is present and behaves correctly.
5. Create some experiences and come back to the simulation panel.
6. Check that we have now two selectors in order to chose between segments and experiences.
7. Check that everything keeps working fine without the feature flag.

Screenshots:
----------------

![Captura de pantalla 2023-06-15 a las 16 20 46](https://github.com/liferay-echo/liferay-portal/assets/3276218/e9f7eec7-edec-48b0-b850-a2fbe72ad0df)
